doc/man7/openssl-env.pod

```diff
@@ -74,6 +74,19 @@ See L<SSL_CTX_load_verify_locations(3)>.
 
 Additional arguments for the L<tsget(1)> command.
 
+=item B<OPENSSL_ia32cap>, B<OPENSSL_sparcv9cap>, B<OPENSSL_ppccap>, B<OPENSSL_armcap>, B<OPENSSL_s390xcap>
```

Comment:
Side note: at some point, we should document those that are currently undocumented

Reply:
Ah, just saw the issues that identify exactly this